packages/cozy-konnector-libs/src/libs/saveFiles.js

```diff
@@ -328,30 +329,21 @@ async function getFileFromMetaData(
   sourceAccountIdentifier,
   slug
 ) {
-  const index = await cozy.data.defineIndex('io.cozy.files', [
-    'metadata.fileIdAttributes',
-    'trashed',
-    'cozyMetadata.sourceAccountIdentifier',
-    'cozyMetadata.createdByApp'
-  ])
-  log(
-    'debug',
-    `Checking existence of ${calculateFileKey(entry, fileIdAttributes)}`
-  )
-  const files = await queryAll(
-    'io.cozy.files',
-    {
-      metadata: {
-        fileIdAttributes: calculateFileKey(entry, fileIdAttributes)
-      },
-      trashed: false,
-      cozyMetadata: {
-        sourceAccountIdentifier,
-        createdByApp: slug
-      }
-    },
-    index
-  )
+  const queryDef = Q(FILES_DOCTYPE)
+    .where({
+      metadata: { fileIdAttributes },
```

Comment:
I thought `fileIdAttributes` was a single attribute, but now I feel like it is actually an object? What is it exactly?

Reply:
No need to use `calculateFileKey(entry, fileIdAttributes)` anymore?